components/WeeklyOverview/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { dummyMedications } from '../../dummydata';
+
+export default function WeeklyOverview() {
+  const weekdays = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+  const supplementsByWeekday = {};
+
+  // Verteile die Supplements auf die Wochentage (1-2 Supplements pro Tag)
+  weekdays.forEach((weekday) => {
+    const supplementsForWeekday = dummyMedications.filter((supplement) => supplement.weekday === weekday);
+    supplementsByWeekday[weekday] = supplementsForWeekday.slice(0, 2);
+  });
```

Comment:
You could build this aggregate with a bit less code and method chaining, if you want:

```suggestion
  weekdays.forEach((weekday) => {
    supplementsByWeekday[weekday] = dummyMedications
      .filter((supplement) => supplement.weekday === weekday)
      .slice(0, 2);
  });
```